ui/src/dataExplorer/components/DashboardsDropdown.tsx

```diff
@@ -58,7 +58,9 @@ class DashboardsDropdown extends PureComponent<Props> {
   }
 
   private get dividerItem(): JSX.Element {
-    return <MultiSelectDropdown.Divider key={0} />
```

Comment:
the default for a new dashboard has `id` of `0` and `key` of `0` so this was causing a key collision